CHANGELOG.md

```diff
@@ -1,5 +1,75 @@
+
+#### 🚀 Updated Proof-of-Spacetime (PoSt)
+
+A new proof construction, [Rational PoSt](https://github.com/filecoin-project/specs/blob/master/proof-of-spacetime.md), has been [implemented](https://github.com/filecoin-project/rust-fil-proofs/pull/763) and [integrated](https://github.com/filecoin-project/go-filecoin/pull/3318). This construction is the same shape as our candidate for testnet and resolves outstanding limitations on proving over many sectors.
+
+#### 🎟️ Block and consensus changes
+
+Block headers are now signed by miners, and election tickets form an array in each header.
+
+#### 🔗 Chain status command
+
+`go-filecoin chain status` is a new command that provides insight into chain sync progress.
+
+### Performance and Reliability
+
+#### ⚡Chain validation performance
+
+Previously in go-filecoin 0.4, we aimed to speed up chain syncing by focusing on the first phase: chain fetching. go-filecoin 0.5 continues that work with improvements to the second phase: chain validation. By switching from HAMT bitwidth 8 to HAMT bitwidth 5, we see a general average improvement in benchmarks of about 4-to-1, across memory usage, speed of operations, and bytes written to disk. Users are encouraged to measure and share their own benchmarks.
+
+### Looking Ahead
+
+#### ✏️ New API design (WIP)
+
+Developers are invited to read and comment on the new [HTTP API design](https://github.com/filecoin-project/filecoin-http-api) (work in progress). This design will be implemented initially in go-filecoin and serve as a standard for interacting with Filecoin nodes across implementations. It will support most of operations offered by the current API and provide a framework for future API growth. 
+
+### User Notes
+
+- The proving period is now configured to 300 rounds (2.5 hrs), down from 1000 rounds (10 hours). We’ve made this temporary change for more frequent node interaction and faster experimentation, and we expect to increase the proving period again in the future.
+- Groth parameters are no longer fetched from the network, but instead locally generated when needed. This can take many minutes (but is more reliable than network).
+- [Block header structure](https://github.com/filecoin-project/go-filecoin/blob/release-0.5.0/types/block.go) has changed, so tools which parse chain data will need updating.
+- The default storage miner waits 15 rounds before beginning a PoSt computation, but is not robust to a subsequent re-org that changes its challenge seed.
+
+### CLI diff
+
+| go-filecoin command | change |
+| ------------------- | ------ |
+| chain status        | added  |
```

Comment:
`mining add-piece` was added, and `mining seal-now` changed slightly. It no longer stages a piece into a sector. It has the same behavior as `--auto-seal-interval-seconds`.

Reply:
Done